macOS/DuckDuckGo/AIChat/Sidebar/AIChatSidebar.swift

```diff
@@ -109,9 +109,6 @@ final class AIChatSidebar: NSObject {
         if let sidebarViewController {
             if persistingState {
                 aiChatURL = sidebarViewController.currentAIChatURL
-                if let restorationData = sidebarViewController.currentAIChatRestorationData {
```

Comment:
No longer needed as restoration data in sidebar data model is wired via publishers